Translations/translation_sk.json

```diff
@@ -5,30 +5,30 @@
 	"messages": {
 		"SettingsCalibrationDone": "Kalibrácia hotová!",
 		"SettingsCalibrationWarning": "Najprv sa prosím uistite, že hrot má izbovú teplotu!",
-		"SettingsResetWarning": "Naozaj chcete obnovit nastavenia?",
+		"SettingsResetWarning": "Naozaj chcete obnovit továrenské nastavenia?",
 		"UVLOWarningString": "Nízke U!",
 		"UndervoltageString": "Nízke napätie",
 		"InputVoltageString": "Vstupné U: ",
 		"WarningTipTempString": "Tep. hrotu: ",
 		"BadTipString": "ZLÝ HROT",
 		"SleepingSimpleString": "Chrr",
-		"SleepingAdvancedString": "Kľudový režim...",
-		"WarningSimpleString": "HOT!",
+		"SleepingAdvancedString": "Pokojový režim..",
+		"WarningSimpleString": "PÁLI",
 		"WarningAdvancedString": "HROT JE HORÚCI !",
 		"SleepingTipAdvancedString": "Hrot:",
 		"IdleTipString": "Hrot:",
-		"IdleSetString": " Set:",
+		"IdleSetString": "Cieľ:",
 		"TipDisconnectedString": "HROT ODPOJENÝ",
 		"SolderingAdvancedPowerPrompt": "Výkon: ",
 		"OffString": "Vyp",
 		"ResetOKMessage": "Reset OK",
 		"YourGainMessage": "Zisk:",
-		"SettingsResetMessage": "Nast. Obnovené!",
-		"NoAccelerometerMessage": "No accelerometer\ndetected!",
-		"NoPowerDeliveryMessage": "No USB-PD IC\ndetected!",
-		"LockingKeysString": " LOCKED",
-		"UnlockingKeysString": "UNLOCKED",
-		"WarningKeysLockedString": "!LOCKED!"
+		"SettingsResetMessage": "Tov. nas. obnov.",
+		"NoAccelerometerMessage": "Chýba akcelerom!",
```

Comment:
I think better will be:
`Chýba pohyb. sens.` or `Chýba pohyb.\nsenzora`, or maybe `Bez` instead of `Chýba`.
(I know it means more like it's error, but would give more

Reply:
I put there ```Bez pohyb. senz.```. There is 16 character length limit.